libs/expo/betterangels/src/lib/ui-components/AddressField/AddressAutocomplete.tsx

```diff
@@ -0,0 +1,136 @@
+import { useApiConfig } from '@monorepo/expo/shared/clients';
+import { AutocompleteInput } from '@monorepo/expo/shared/ui-components';
+import { debounce } from '@monorepo/expo/shared/utils';
+import { useCallback, useState } from 'react';
+import { Control, Controller, FieldValues, Path } from 'react-hook-form';
+import {
+  TPlaceResult,
+  TPlacesPrediction,
+  getPlaceAutocomplete,
+  getPlaceDetailsById,
+} from '../../services';
+import { AddressOption } from './AddressOption';
+
+const DEFAULT_DEBOUNCE_MS = 300;
```

Comment:
@vecchp -  I'd suggest changing `300ms` timeout to `100ms`
300 is annoying, and I'd even go 50ms until we actually notice a pricing issue.

If it becomes expensive then perhaps that's a good sign.
I think it's 2.83 USD per 1000 per [this pricing page](https://developers.google.com/maps/documentation/places/web-service/usage-and-billing), but **** google and their docs 🤷